README.md

```diff
@@ -65,6 +65,105 @@ or
 import worker from 'workerize-loader?inline!./worker'
 ```
 
+#### `name`
+
+Type: `String`
+Default: `[hash]`
+
+You can pass filename prefix for the compiled worker. Note, that `.worker` suffix will be added automatically
+
+```js
+// webpack.config.js
+{
+  loader: 'workerize-loader',
+  options: { name: '[name].[contenthash:8]' }
+}
+```
+or 
+```js
+import worker from 'workerize-loader?{"name":"[name].[contenthash:8]"}!./worker'
+```
+
+#### `publicPath`
+
+Type: `String`
+Default: based on `output.publicPath`
+
+You can pass `publicPath` to specify the public URL address of the output files
+
+```js
+// webpack.config.js
+{
+  loader: 'workerize-loader',
+  options: { publicPath: '/static/' }
+}
+```
+or 
+```js
+import worker from 'workerize-loader?{"publicPath":"/static/"}!./worker'
+```
+
+#### `ready`
+
+Type: `Boolean`
+Default: `false`
+
+You can pass `ready` to set up a wrapper around the `"ready"` event
```

Comment:
```suggestion
If `true`, the imported "workerized" module will include a `ready` property, which is a Promise that resolves once the Worker has been loaded. Note: this is unnecessary in most cases, since worker methods can be called prior to the worker being loaded.
```